CI Build Failure after updating to Xcode 12

### Description

After upgrading to Xcode 12, I'm seeing the following build failure from the commandline:

```
$ tuist build
❌ error: The linked and embedded framework 'Kite.framework' is missing one or more architectures required by this target: arm64. (in target 'App' from project 'App')
The 'xcodebuild' command exited with error code 65 and message:
** BUILD FAILED **
```

The build succeeds when run from Xcode.

### Expected

The `tuist build` command should pass.

--- a/.github/workflows/ios.yml
+++ b/.github/workflows/ios.yml
@@ -6,6 +6,9 @@ on:
   pull_request:
     branches: [ master ]
 
+env:
+  DEVELOPER_DIR: /Applications/Xcode_12.app/Contents/Developer
+
 jobs:
   build:
     name: Build default scheme
@@ -17,7 +20,12 @@ jobs:
       - name: Install Tuist
         run: |
           bash <(curl -Ls https://install.tuist.io)
-      - name: Build
+      - name: Setup
         run: |
           tuist up
-          tuist build
+      - name: Build Frameworks
+        run: |
+          tuist build Kite
+      - name: Build App
+        run: |
+          tuist build App

